win32ss/gdi/ntgdi/freetype.c

```diff
@@ -3002,6 +2998,17 @@ IntRequestFontSize(PDC dc, FT_Face face, LONG Width, LONG Height)
         Height = Width;
     }
 
+    if (Height < 0)
+    {
+        /* case (A): lfHeight is negative: the character height */
+        Height = -Height;
+    }
+    else if (Height > 0)
+    {
+        /* case (B): lfHeight is positive: the cell height */
+        Height = Height * face->units_per_EM / face->height;
```

Comment:
If Height==0 and face->height==0 wouldn't that crash? Or if face==NULL?

Reply:
If face is NULL, FT_New_Memory_Face will fail. I think it doesn't come here in the case.
I can't deny the case that face->height can be zero.
I will add ASSERT(face != NULL); and ASSERT(face->height != 0);.
And if face->height == 0, then I will use height value 1.

Reply:
Done.